libsrc/dataset.c

```diff
@@ -152,6 +152,11 @@ static int delete_var(const ISMRMRD_Dataset *dset, const char *var) {
     return status;
 }
 
+#define ISMRMRD_READ_BUFFER_SIZE 1024*1024
```

Comment:
How did you arrive at this number?

Reply:
It's the same as the default value in HDF5. 
I haven't played around with it too much, but 1024 seemed to give some degradation in write performance on linux.